spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -83,30 +88,43 @@ def get_all_ocean_cluster(self):
 
         return formatted_response["response"]["items"]
 
-    def get_all_ocean_sizing(self, ocean_id, namespace):
+    def get_all_ocean_sizing(self, ocean_id: str, filter: aws_ocean.RightSizingRecommendationFilter):
```

Comment:
Lets not change this function. It may be used by some customers.

Reply:
Done